docs/carts/cart_interactions/add_packages.md

```diff
@@ -17,6 +17,9 @@ This will add all of the package's included items to the cart.
 | ---------------------------------- | ------------------------------------------------------ |
 | `package[id]`                      | The ID of the package to add to the cart.              |
 | `package[quantity]`                | The number of the packages to add to the cart at once. |
+| `package[stat_date]`               | The customers selected start date for the package.     |
```

Comment:
Is this only relevant for flexible date packages?

Reply:
Yes, have updated the copy to reflect this.